### DISCUSSION TITLE
How do I update a yarn plugin, e.g. how do I update the interactive-tools plugin?

### DISCUSSION THREAD

**Discussion by Axedyson** [yarnpkg/berry#4823] [Q&A] [answered]
What the title says. Is the way to update the plugin simply by downloading it again, like this: `yarn plugin import interactive-tools`? Also how do I even know if there are any new updates to the plugin in the first place?

**Comment by RDIL** [comment, marked as answer]
At present you just re-import the plugin.  It's not the greatest in my opinion, but it is the current way it's done.  In Yarn 4, the core plugins like interactive-tools are bundled into the actual Yarn build, so this should hopefully reduce the friction on that front.

**Reply by pastelmind** [reply]
This needs to be documented better. It should be mentioned in the [Plugin Tutorial](https://yarnpkg.com/advanced/plugin-tutorial), not buried in a GitHub discussion.

**Reply by RDIL** [reply]
Feel free to open an issue or a PR for it!
